pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraDescription.scala

```diff
@@ -12,50 +12,37 @@ object SierraDescription extends SierraDataTransformer with SierraQueryOps {
 
   // Populate wwork:description.
   //
-  // We use MARC field "520"
+  // We use MARC field "520".  Rules:
   //
-  // The value comes from comes subfield $a concatenated with subfield $b.
+  //  - Join 520 ǂa and ǂb with a space
+  //  - Wrap resulting string in <p> tags
+  //  - Join each occurrence of 520 into description
   //
   // Notes:
-  //  - A bib may have multiple 520 records, in which case we join with spaces
-  //  - If $b is empty, we just use $a
-  //  - We never expect to see a record with $b but not $a
+  //  - Both ǂa (summary) and ǂb (expansion of summary note) are
```

Comment:
What are these symbols? :thinking:  https://en.wikipedia.org/wiki/Palatal_click

Reply:
I’ve seen them used in MARC reference docs for subfield tags. I’m not sure there’s a consistent standard. 🤷‍♀️